### DISCUSSION TITLE
Share ideas for new features

### DISCUSSION THREAD

**Discussion by rohitsangwan01** [rohitsangwan01/uni_control_hub#26] [Ideas]
Share your Ideas for new Features here

**Comment by CRdeGraaf** [comment]
the heart of this app (mouse and keyboard) works so much better than any other project. Its practically native when connecting android to a macbook. I am seriously impressed.

The only thing that keeps productivity down is the absolute/relative mouse position messing with the viewport size. the server app is trying to simulate where the mouse could possibly be on the tablet. I understand the need to workaround this because everyone wants to trigger device switching upon touching the side of the device. Even when i turn off mouse-precision on the android tablet, the mouse position in the viewport just isnt reliably tracking.

With a small change i think it could work very reliably: approach the core system more like a KVM rather than multi-monitor setups.

If you can let users turn off screen-edge-switching and bind device switching to something like opt+cmd+1, opt+cmd+2, etc, then the experience is fully equivalent to passthrough functionality of a keyboard and mouse to another device over usb. Which is an amazing use case on its own.

**Reply by rohitsangwan01** [reply]
Hey thank you for the feedback, you can achieve something similar in current version as well,
From app settings you can configure shortcut to lock the cursor in specific device

**Reply by d4nnyc4ts** [reply]
i came here to ask about locking the mouse to a screen as well as inverting the scroll wheel on android only to find out i have missed a setting menu in the app and it was already taken care of. this is an absolutely brilliant app Thanks for your hard work.
